### PR TITLE
Gh 2767/create delegate key for all key types

### PR DESCRIPTION
**Pull request by MouazAlzahabi**
Handles #2767

### CODE REVIEW

File: Multisig/Cross-layer/Analytics/TrackingEvent.swift [hunk 1]
```diff
@@ -259,7 +259,10 @@ enum TrackingEvent: String, Trackable {
     // MARK: add delegate key
     case addDelegateKeyLedger                       = "screen_delegate_ledger_nano_x"
     case addDelegateKeyWalletConnect                = "screen_delegate_walletconnect"
-
+    case addDelegateKeyImported                     = "screen_delegate_imported"
+    case addDelegateKeyGenerated                    = "screen_delegate_generated"
+    case addDelegateKeyKeystone                     = "screen_delegate_keystone"
```
Comment (DmitryBespalov) [inline Multisig/Cross-layer/Analytics/TrackingEvent.swift]: @MouazAlzahabi this didn't come in the requirements, however, can we join these 3 tracking events into one, and track it under `screen_add_delegate`, and have a parameter `key_type` with one of 3 values `imported`, `generated`, `keystone`, and `walletconnect`, and `ledger`?

Comment (DmitryBespalov) [reply Multisig/Cross-layer/Analytics/TrackingEvent.swift]: (in another refactoring)

Comment (DmitryBespalov) [reply Multisig/Cross-layer/Analytics/TrackingEvent.swift]: actually, never mind, maybe we make a separate refactoring of tracking.

File: Multisig/UI/Settings/OwnerKeyManagement/KeyAddedViewController.swift [hunk 2]
```diff
@@ -1,22 +1,26 @@
 //
-//  WalletConnectKeyAdded.swift
+//  KeyAddedViewController.swift
 //  Multisig
 //
-//  Created by Moaaz on 11/22/21.
-//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//  Created by Mouaz on 11/1/22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/OwnerKeyManagement/KeyAddedViewController.swift:6]: Time goes by...

File: Multisig/Cross-layer/Analytics/TrackingEvent.swift (hunk 1 above)
Comment (MouazAlzahabi) [reply Multisig/Cross-layer/Analytics/TrackingEvent.swift]: I made it like this because we already had two for WalletConnect&ledger

**Review by DmitryBespalov — APPROVED** [review]
Good!

File: Multisig/Cross-layer/Analytics/TrackingEvent.swift (hunk 1 above)
Comment (MouazAlzahabi) [reply Multisig/Cross-layer/Analytics/TrackingEvent.swift]: @DmitryBespalov done

**Review by elgatovital — APPROVED** [review]

**Pull request merged**